config/splunk/docker-compose.yml

```
@@ -0,0 +1,13 @@
+---
+version: '3.1'
+services:
+  splunk:
+    image: splunk/splunk
```

Please be explicit with the registry:

```suggestion
    image: docker.io/splunk/splunk
```